Error in readChar(con, 5L, useBytes = TRUE) : cannot open the connection
In addition: Warning message:
In readChar(con, 5L, useBytes = TRUE) :
  cannot open compressed file '2110/rdas/compare-baseline-cross_verificar.RData', probable reason 'No such file or directory'
Loading required package: lattice
Loading required package: ggplot2
Loading required package: doMC
Loading required package: foreach
Loading required package: iterators
Loading required package: parallel
Loading required package: kernlab

Attaching package: ‘kernlab’

The following object is masked from ‘package:ggplot2’:

    alpha

Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
Warning message:
In .local(x, ...) : Variable(s) `' constant. Cannot scale data.
